tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -0,0 +1,285 @@
+import * as React from 'react';
+
+import { fetchGroupedRatings, setGroupedRatings } from 'amo/actions/reviews';
+import RatingsByStar, {
+  extractId,
+  RatingsByStarBase,
+} from 'amo/components/RatingsByStar';
+import { ErrorHandler } from 'core/errorHandler';
+import { createInternalAddon } from 'core/reducers/addons';
+import { fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+import { dispatchClientMetadata, fakeAddon } from 'tests/unit/amo/helpers';
+import ErrorList from 'ui/components/ErrorList';
+import LoadingText from 'ui/components/LoadingText';
+
+describe(__filename, () => {
+  let store;
+
+  beforeEach(() => {
+    store = dispatchClientMetadata().store;
+  });
+
+  const getProps = ({
+    addon = createInternalAddon(fakeAddon),
+    ...customProps
+  } = {}) => {
+    return { addon, i18n: fakeI18n(), store, ...customProps };
+  };
+
+  const render = (customProps = {}) => {
+    const props = getProps(customProps);
+
+    return shallowUntilTarget(<RatingsByStar {...props} />, RatingsByStarBase);
+  };
+
+  const addonForGrouping = (grouping, addonParams = {}) => {
+    return createInternalAddon({
+      ...fakeAddon,
+      ratings: {
+        ...fakeAddon.ratings,
+        // This is the sum of all grouped rating counts.
+        count: Object.values(grouping).reduce(
+          (total, currentValue) => total + currentValue,
+        ),
+      },
+      ...addonParams,
+    });
+  };
+
+  const createErrorHandlerWithError = () => {
+    const errorHandler = new ErrorHandler({
+      id: 'some-id',
+      dispatch: store.dispatch,
+    });
+    errorHandler.handle(new Error('some unexpected error'));
+    return errorHandler;
```

Comment:
If that is slightly better than using `createStubErrorHandler()`, I think we should change this helper function (assuming the main concern is injecting the `capturedError` directly, without using `handle()`).